### PR TITLE
Create olegg69-hometown

### PR DESCRIPTION
**Pull request by olegg69**
## Checking first time.

### CODE REVIEW

**Comment by jwager0408** [comment]
any places to eat

File: olegg69-hometown [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Castro Valley, CA
```
Comment (nathyga) [inline olegg69-hometown]: Maybe you should add a heading of places to visit. What can i visit in your hometown?

File: olegg69-hometown [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+### Great place to live
+
```
Comment (olegg69) [inline olegg69-hometown]: - Good climate
- Awesome views

**Comment by olegg69** [comment]
Reopening the branch.

**Pull request merged**